.github/workflows/npm.yml

```diff
@@ -11,7 +11,7 @@ on:
       - master
 
 env:
-  NODE_VERSION: "10.x" # set this to the node version to use
+  NODE_VERSION: "12.x" # set this to the node version to use
```

Comment:
Updated to 12.x since there will be things we will start using that aren't supported in 10.x.